pkg/shell/topnav.jsx

```diff
@@ -252,7 +252,7 @@ export class TopNav extends React.Component {
                             { this.props.index.has_oops &&
                                 <ToolbarItem>
                                     <Button id="navbar-oops" variant="link" size="lg" isDanger
-                                            onClick={() => Dialogs.show(<OopsModal />)}>{_("Ooops!")}</Button>
+                                            onClick={() => Dialogs.run(OopsModal, {})}>{_("Ooops!")}</Button>
```

Comment:
This added line is not executed by any test.

Reply:
Interesting :mag: